<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Icon/Bokehlicia-Captiva-Rocket.ico</normaloff>../Icon/Bokehlicia-Captiva-Rocket.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pb_Account">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ACCOUNT</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Graphicloads-Flat-Finance-Lock.ico</normaloff>../Icon/Graphicloads-Flat-Finance-Lock.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
          <property name="default">
           <bool>false</bool>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_whitelist">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>WHITELIST</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Papirus-Team-Papirus-Places-Folder-green-activities.ico</normaloff>../Icon/Papirus-Team-Papirus-Places-Folder-green-activities.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Follow">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>FOLLOW</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Graphicloads-100-Flat-2-People.ico</normaloff>../Icon/Graphicloads-100-Flat-2-People.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Dm">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>DM</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Graphicloads-100-Flat-2-Email.ico</normaloff>../Icon/Graphicloads-100-Flat-2-Email.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Like">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LIKE</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Double-J-Design-Origami-Colored-Pencil-Red-heart.ico</normaloff>../Icon/Double-J-Design-Origami-Colored-Pencil-Red-heart.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pb_Limit">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>LIMIT</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Bokehlicia-Captiva-Preferences.ico</normaloff>../Icon/Bokehlicia-Captiva-Preferences.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_blacklist">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>BLACKLIST</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Papirus-Team-Papirus-Places-Folder-red-activities.ico</normaloff>../Icon/Papirus-Team-Papirus-Places-Folder-red-activities.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Unfollow">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>UNFOLLOW</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Oxygen-Icons.org-Oxygen-Actions-list-remove-user.ico</normaloff>../Icon/Oxygen-Icons.org-Oxygen-Actions-list-remove-user.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Comment">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>COMMENT</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Graphicloads-100-Flat-2-Chat-2.ico</normaloff>../Icon/Graphicloads-100-Flat-2-Chat-2.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Reset">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RESET</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Bokehlicia-Captiva-Deja-dup.ico</normaloff>../Icon/Bokehlicia-Captiva-Deja-dup.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Running:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Follow</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="text">
         <string>Unfollow</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="text">
         <string>Comment</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_5">
        <property name="text">
         <string>Like</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="whatsThis">
         <string/>
        </property>
        <property name="text">
         <string>Direct Message</string>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:600; color:#ff007f;&quot;&gt;Welcome to Virtual Mode&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Enjoy Your Fully Secured Automation Software&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="pb_Stop">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Graphicloads-100-Flat-Stop-2.ico</normaloff>../Icon/Graphicloads-100-Flat-Stop-2.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_Start">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icon/Graphicloads-100-Flat-Play.ico</normaloff>../Icon/Graphicloads-100-Flat-Play.ico</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="autoDefault">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLast_Activities"/>
    <addaction name="actionClear_Log"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionRenew_License"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionTerms"/>
    <addaction name="actionAbout_Virtual_Mode"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionRenew_License">
   <property name="icon">
    <iconset>
     <normalon>../../../.designer/backup/Icon/Graphicloads-100-Flat-2-Arrow-refresh-4.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Renew License</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset>
     <normalon>../../../.designer/backup/Icon/Hopstarter-Sleek-Xp-Basic-Close.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset>
     <normalon>../../../.designer/backup/Icon/Icons8-Windows-8-User-Interface-Logout.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
  <action name="actionTerms">
   <property name="icon">
    <iconset>
     <normalon>../../../.designer/backup/Icon/Hopstarter-Sleek-Xp-Basic-Document-Write.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Terms</string>
   </property>
  </action>
  <action name="actionAbout_Virtual_Mode">
   <property name="icon">
    <iconset>
     <normalon>../../../.designer/backup/Icon/Oxygen-Icons.org-Oxygen-Actions-help-about.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>About Virtual Mode</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionLast_Activities">
   <property name="text">
    <string>Last Activities log</string>
   </property>
  </action>
  <action name="actionClear_Log">
   <property name="text">
    <string>Clear Log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClear_Log</sender>
   <signal>triggered()</signal>
   <receiver>textEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
